src/cbang/openssl/SSLContext.cpp

```diff
@@ -291,8 +323,120 @@ void SSLContext::loadSystemRootCerts() {
   CertCloseStore(store, 0);
 
 #elif __APPLE__
-  // TODO
-  LOG_WARNING(__FUNCTION__ << "() Not yet implemented on OSX");
+  X509_STORE *verifyStore = SSL_CTX_get_cert_store(ctx);
+
+  OSStatus err;
+  CFArrayRef anchors = NULL;
+  CFIndex i, count;
+  CFIndex trustedCount = 0;
```

Comment:
It's more C++y to put these type defs where the variable is first used.  Or in the case of an index like ``i`` in the for loop.  E.g. ``for (CFIndex i = 0; i < count; i++) {``.

Reply:
Ok. I Think I moved all I could.